website/blog/2022/04/29/interview with ju zhiyuan: becoming one of the 918 asf members, i felt thrilled and proud

```diff
@@ -0,0 +1,181 @@
+---
+title: "Interview with JU Zhiyuan: Becoming One of the 918 ASF Members, I Felt Thrilled and Proud"
+slug: /blog/2022/04/29/interview-juzhiyuan-apache-member
+authors:
+  - name: "Yahui Rao"
+    title: "Author"
+    url: "https://github.com/EstelleRao"
+    image_url: "https://github.com/EstelleRao.png"
+keywords:
+- APISIX
+- Apache APISIX
+- API Gateway
+- ASF Member
+- Apache Member
+- Open Source
+- Remote Work
+- Full-Stack Web Developer
+description: In this interview, we invited JU Zhiyuan, a PMC member of Apache APISIX, to share his feeling about becoming an ASF member, his experience with open source, and his self-learning journey in technology.
+tags: [Interview]
+---
+
+> JU Zhiyuan, a PPMC member of Apache APISIX, was elected to be one of the 918 ASF Members in March 2022. In this interview, Zhiyuan shared his experience with open source and his self-learning journey in technology.
```

Comment:
```suggestion
> Ju Zhiyuan, a PPMC member of Apache APISIX, was elected to be one of the 918 ASF Members in March 2022. In this interview, Zhiyuan shared his experience with open source and his self-learning journey in technology.
```

Reply:
I prefer writing a Chinese name in this way for several reasons. 
First, I studied a lot of materials about how to write a Chinese name in a standard way but found no widely recognized standard. Chinese names are written in several formats, including Han Meimei, Meimei Han, and HAN Meimei. 
Second, as the tradition of Chinese names is different from others and now familiar to other countries, I prefer writing them in this traditional way but capitalizing the family name to make it more clear.